engine/src/main/java/org/terasology/engine/core/PathManager.java

```diff
@@ -67,7 +65,7 @@ public final class PathManager {
 
     private PathManager() {
         installPath = findInstallPath();
-        homePath = installPath;
+        homePath = PROJECT_PATH;
```

Comment:
```suggestion
        homePath = installPath;
```
`findInstallPath()` provides a fallback path for unusual scenarios that I think we should keep. Defaulting to the current directory is normal for development anyway, I think. You can default to `PROJECT_PATH` in `findInstallPath` if needed.

Reply:
returning projectpath in a function called findinstallpath i thought is too much :) but let me digest your suggestion ...